.github/CODEOWNERS

```diff
@@ -1,6 +1,7 @@
 # owners and reviewers
-configs             @Alexsandruss @samir-nasibli @napetrov
-sklbench            @Alexsandruss @samir-nasibli
-*.md                @Alexsandruss @napetrov
-requirements*.txt   @Alexsandruss @napetrov
-conda-env-*.yml     @Alexsandruss @napetrov
+configs             @Alexsandruss

```

Comment:
not critical for merge , but i would also put Ethan as owner on requerements/env as well at least. Ideally we need 2+ owners for other entries as well

Reply:
Added Ethan. I guess code owners list will grow with adoption of refactored version of benchmarks.